### DISCUSSION TITLE
How to do persistent relational projections

### DISCUSSION THREAD

**Discussion by aef** [RailsEventStore/rails_event_store#1134] [General]
I asked my [question](https://stackoverflow.com/q/56974095/322863) over on StackOverflow. I would appreciate help on finding an answer.

Thanks.

**Comment by mostlyobvious** [comment]
I've posted reply on SO. Feel free to comment there further or reopen this issue.

PS. I've added `rails-event-store` tag to my watched tags not to miss any further RES-related topics
